source/docs/software/pathplanning/system-identification/viewing-diagnostics.rst

```diff
@@ -1,53 +1,63 @@
 Viewing Diagnostics
 ===================
 
-The first diagnostic to look at is the r-squared - it should be somewhere north of ~.9. If it is significantly lower than this, there is likely a problem with your identification data.
+Goodness-of-Fit Metrics
+-----------------------
+
+There are three numerical accuracy metrics that are computed with this tool: acceleration r-squared, simulated velocity r-squared, and RMSE.
+
+.. image:: images/viewing-metrics.png
+   :alt: Analysis accuracy metrics
+
+The `r-squared <https://en.wikipedia.org/wiki/Coefficient_of_determination>`__ metric refers to the goodness of fit of the tool's `OLS (Ordinary Least Squares) fit <https://en.wikipedia.org/wiki/Ordinary_least_squares>`__.  The `RMSE (Root Mean Squared Error) <https://en.wikipedia.org/wiki/Root-mean-square_deviation>`__ is the error (in the measured units) between the collected velocities and the predicted velocities from a simulation based on the feedforward gains.
 
-The next diagnostic to look at is the RMSE to make sure that the error is reasonable and within your desired constraints.
+The acceleration r-squared is the fraction of the variance in measured acceleration (used as the independent variable in the SysId regression) explained by the linear model.  This is usually quite low, because acceleration is very susceptible to system noise.
 
-To investigate further, you can look at several diagnostic plots on the side.
+The simulated velocity r-squared is the fraction of the variance in measured velocity explained by a noiseless simulation of the motor movement stepped forward with the constants determined from the regression.  A value north of ``.9`` indicates a good fit.
+
+The simulated velocity RMSE is the standard deviation of the velocity error from the simulated model.  This is a good estimation of the amount of process noise present in your system, and can be used as a starting guess for the model noise term in :ref:`state-space control <docs/software/advanced-controls/state-space/state-space-intro:Introduction to State-Space Control>` - remember, though, that there may be more external forces to handle in actual operation than during your test routine.
```

Comment:
Seems like a run-on but I am not the grammar man so I defer to @Daltz333 

```suggestion
The simulated velocity RMSE is the standard deviation of the velocity error from the simulated model.  This is a good estimation of the amount of process noise present in your system.  It can then be used as a starting guess for the model noise term in :ref:`state-space control <docs/software/advanced-controls/state-space/state-space-intro:Introduction to State-Space Control>`.  Remember that there may be more external forces to handle in actual operation than during your test routine.
```

Reply:
Jason's suggestion seems appropriate.